packages/graphback/src/GraphQLBackend.ts

```diff
@@ -89,10 +94,24 @@ export class GraphQLBackendCreator {
       // logger.error(`Error on Database creation ${error}`)
       throw error
     }
+  }
 
+  /**
+   * Filters duplicate fields (by name) from a list of Fields
+   *
+   * @private
+   * @param {Field[]} fields The original list of fields from the type
+   * @returns {Field[]} Return the filtered list of fields
+   * @memberof GraphQLBackendCreator
+   */
+  private getUniqueFields(fields: Field[]): Field[] {
```

Comment:
Question - why inputContext will have duplicates at this point?

Do we need to handle cases when someone shadows the same field with different type? Which one wins now interface or direct definition? This code looks like direct definition will be omited.

Reply:
At this point, inputContext should not have duplicates. Looking to change this now.

We need to see which field takes precedence when both are the same.